opensrp-app/src/main/java/org/smartregister/ssl/OpensrpSSLHelper.java

```diff
@@ -13,17 +13,26 @@
 import java.io.InputStream;
 import java.security.KeyStore;
 
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
 import javax.net.ssl.SSLException;
+import javax.net.ssl.TrustManagerFactory;
 
 /**
  * Created by onaio on 01/09/2017.
  */
 
 public class OpensrpSSLHelper {
+
+    private static final String X509 = "X509";
+    private static final String TLS = "TLS";
+    private static final String PASS = "phone red pen";
```

Comment:
Could this potentially be converted into a configuration, passed to the app during compilation?

Reply:
yes that is possible

Reply:
This would make it difficult to enforce security standards

Reply:
These constants have been moved to build configuration